.github/workflows/test.yml

```diff
@@ -143,6 +143,8 @@ jobs:
             flags: -D UNSIGNED_CHAR=ON
           - cc: clang
             runner: macos-12
+            flags: -D CMAKE_FIND_FRAMEWORK=LAST
+            deps_flags: -D CMAKE_FIND_FRAMEWORK=LAST
```

Comment:
Can this be in the cmakefiles instead? We should try to avoid CI-provider specific solutions, and bake in the solutions to the build system if possible.